node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala

```diff
@@ -174,6 +184,34 @@ class UtilsRouteSpec extends RouteSpec("/utils") with RestAPISettingsHelper with
 
+  val dApp =
+    """
+      |{-# STDLIB_VERSION 3 #-}
+      |{-# CONTENT_TYPE DAPP #-}
+      |{-# SCRIPT_TYPE ACCOUNT #-}
+      |
+      |@Callable(i)
+      |func write(value: Int) = {
+      |    WriteSet([DataEntry("result", value)])
+      |}
+      |
+      |@Callable(i)
+      |func sendAsset(recipient: String, amount: Int, assetId: String) = {
+      |    TransferSet([ScriptTransfer(Address(recipient.fromBase58String()), amount, assetId.fromBase58String())])
+      |}
+      |
+      |@Callable(i)
+      |func writeAndSendWaves(value: Int, recipient: String, amount: Int) = {
+      |    ScriptResult(
+      |        WriteSet([DataEntry("result", value)]),
+      |        TransferSet([ScriptTransfer(Address(recipient.fromBase58String()), amount, unit)])
+      |    )
+      |}
+      |
+      |@Verifier(tx)
```

Comment:
also check explicitly that without the verifier function the `complexity` field is 0

Reply:
Ok